build.rs

```diff
@@ -110,18 +126,20 @@ fn make_source() {
     // When building libusb from source, allow use of its logging facilities to aid debugging.
     // FIXME: This does not link correctly under MinGW due to a rustc bug, so only do it on MSVC
     // Ref: https://github.com/rust-lang/rust/issues/47048
-    if cfg!(target_env = "msvc") {
+    if std::env::var("CARGO_CFG_TARGET_ENV").as_ref().map(|s| s.as_ref()) == Ok("msvc") {
```

Comment:
```suggestion
    if std::env::var("CARGO_CFG_TARGET_ENV") == Ok("msvc".into()) {
```
Looks better? And it other places too.

Reply:
@a1ien I made the updates, cleaned, rebuilt, and got a valid Android artifact on Windows.